packages/wallets/src/components/FlowField/FlowTextField.tsx

```diff
@@ -57,7 +57,12 @@ const FlowTextField = forwardRef(
                             errorMessage={hasTouched && (form.errors[name] || errorMessage)}
                             isInvalid={(hasTouched && isInvalid) || (hasTouched && Boolean(form.errors[name]))}
                             name={field.name}
-                            onChange={field.onChange}
+                            onChange={e => {
+                                field.onChange(e);
+                                if (rest.onChange) {
+                                    rest.onChange(e);
```

Comment:
```suggestion
                                rest?.onChange(e))
```

Reply:
This is not needed anymore I will be removing this but thanks for the suggestions 👍🏼